app/controllers/admins/employees_controller.rb

```diff
@@ -27,6 +27,31 @@ def destroy
       redirect_back fallback_location: kudos_path
     end
 
+    def add_kudos_form
+      render 'add_kudos_form'
+    end
+
+    def add_kudos
+      kudos_number = add_kudos_params[:number_of_available_kudos_to_add].to_i
+      unless kudos_number.between?(1, 20)
+        flash[:alert] = 'Number of additional kudos must be in range 1 to 20.'
+        render 'add_kudos_form'
+        return
+      end
+
+      Employee.transaction do
+        Employee.find_each do |employee|
+          employee.update!(number_of_available_kudos: employee.number_of_available_kudos + kudos_number)
+        end
+      end
+    rescue ActiveRecord::RecordInvalid => e
+      flash[:alert] = e
```

Comment:
do not show error messages to the user :) It can simply say - updating user has failed or something like that